client/blocks/reader-post-card/style.scss

```diff
@@ -209,12 +209,6 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 			color: $gray;
 		}
 
-		.reader-post-options-menu {
```

Comment:
Good call. I added this back: https://github.com/Automattic/wp-calypso/pull/17070/commits/3c01dedc85883695bdad8261e12bf2d441b4470b